index.test.js

```diff
@@ -178,4 +182,38 @@ describe("raven-for-redux", function() {
       );
     });
   });
+
+  describe("with filterBreadcrumbActions option enabled", function() {
+    beforeEach(function() {
+      context.filterBreadcrumbActions = action => {
+        return action.type !== "UNINTERESTING_ACTION";
+      };
+
+      context.store = createStore(
+        reducer,
+        applyMiddleware(
+          createRavenMiddleware(Raven, {
+            filterBreadcrumbActions: context.filterBreadcrumbActions
+          })
+        )
+      );
+    });
+    it("filters actions for breadcrumbs", function() {
+      context.store.dispatch({ type: "INCREMENT" });
+      context.store.dispatch({ type: "UNINTERESTING_ACTION" });
+      context.store.dispatch({ type: "INCREMENT" });
+      context.store.dispatch({ type: "UNINTERESTING_ACTION" });
+      context.store.dispatch({ type: "UNINTERESTING_ACTION" });
+      Raven.captureMessage("report!");
+
+      expect(context.mockTransport).toHaveBeenCalledTimes(1);
+      const {
+        extra,
+        breadcrumbs
+      } = context.mockTransport.mock.calls[0][0].data;
+      // Even though the action isn't added to breadcrumbs, it should be sent with extra data
```

Comment:
I think this would be better as a separate `it()` block, that way if it fails, we get a nice error message, but don't worry about it. I can always clean this up later.